.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "deno.enable": true,
+  "deno.lint": true,
+  "deno.unstable": false
+}
```

Comment:
Remove this file... Maybe we should add it to .gitignore.

Reply:
woops, i did remove it, jsut forgot to push

Reply:
https://github.com/drashland/deno-drash/pull/498/commits/864facaa65d6c7423eea972055fcc4b6335f65c7